utility/mirrormanager2.spec

```diff
@@ -23,7 +19,7 @@ Summary:        Mirror management application
 # to generate the worldmaps are licensed under GPLv2 and GPLv2+
 License:        MIT and GPLv2+ and GPLv2
 URL:            https://github.com/fedora-infra/mirrormanager2/
-Source0:        %{url}/archive/%{version}/%{name}-%{version}.tar.gz
+Source0:        mirrormanager2-0.17.tar.gz
```

Comment:
Hardcoding the version seems unnecessary.

Reply:
Mmm...I think it might have been initiative of packit...changing back